### PR TITLE
removed constantvelocityacquisition from codebase

### LINKED ISSUE #894: Move constant velocity acquisition mode to a plugin
...

### PR DESCRIPTION
**Pull request by mehr0096**

### CODE REVIEW

File: src/navigate/model/metadata_sources/metadata.py [hunk 1]
```diff
@@ -232,18 +228,11 @@ def set_stack_order_from_configuration_experiment(self) -> None:
...
-                and state["image_mode"] == "confocal-projection"
-            )
-            or (
-                state["stack_cycling_mode"] == "per_stack"
-                and state["image_mode"] == "ConstantVelocityAcquisition"
-            )
+            state["stack_cycling_mode"] == "per_stack"
+            and state["image_mode"] == "z-stack"
+        ) or (
+            state["conpro_cycling_mode"] == "per_stack"
```
Comment (AdvancedImagingUTSW) [inline src/navigate/model/metadata_sources/metadata.py:234]: Is this necessary now that the confocal projection mode is a plugin?

**Review by AdvancedImagingUTSW — COMMENTED** [review]
I have tested this and updated the documentation. The only thing that is not clear is if the metadata needs to keep the confocal projection mode...

**Comment by mehr0096** [comment]
I wasn't sure if that was part of the functionality of the confocal projection mode plugin so I didn't remove anything anything associated with confocal projection.

File: src/navigate/model/metadata_sources/metadata.py (hunk 1 above)
Comment (AdvancedImagingUTSW) [reply src/navigate/model/metadata_sources/metadata.py:234]: Created a separate issue. Moving forward with accepting and merging.

**Review by AdvancedImagingUTSW — APPROVED** [review]

**Pull request merged**